### PR TITLE
create-jar-details

### PR DESCRIPTION
**Pull request by Dmytro-Foliushniak**

### CODE REVIEW

File: frontend/mono_frontend/src/pages/JarDetails/JarDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+			)
+		}, 100)
+
+		return null
+	}
+}
+
+export const JarDetails = function () {
+	const jar = useRouteLoaderData('jar')
+	console.log(1)
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/pages/JarDetails/JarDetails.jsx:36]: leftover

File: frontend/mono_frontend/src/utils/convertProgressInPercent.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const convertProgressInPercent = function (balance, goal) {
+	return String(Math.round((balance / goal) * 100) + '%')
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/utils/convertProgressInPercent.js:2]: Potential bug:
зображення

Comment (Iron-Cow) [reply frontend/mono_frontend/src/utils/convertProgressInPercent.js:2]: Ideally - create unit tests for all utils

File: frontend/mono_frontend/src/components/__tests__/JarDetails.test.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+}))
+jest.mock('../../utils/convertProgressInPercent', () => ({
+	convertProgressInPercent: jest.fn()
+}))
+
+describe('JarDetails component', () => {
+	const jarData = {
+		id: 1,
+		title: 'Test Jar',
+		balance: 5000,
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/components/__tests__/JarDetails.test.js:22]: Lets change it to something else to get result different to `50%`  (to not confuse with jar without goal)

Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/components/__tests__/JarDetails.test.js:22]: Done

File: frontend/mono_frontend/src/utils/convertProgressInPercent.js (hunk 2 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/utils/convertProgressInPercent.js:2]: Done

File: frontend/mono_frontend/src/pages/JarDetails/JarDetails.jsx (hunk 1 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/pages/JarDetails/JarDetails.jsx:36]: Done

**Review by Iron-Cow — APPROVED** [review]

**Pull request merged**